pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -13,14 +15,26 @@
 
 from pulp_smash.api import _get_sleep_time
 from pulp_smash.config import get_config
+from pulp_smash import cli as smash_cli
 from pulp_smash.pulp3.bindings import delete_orphans, monitor_task
 from pulp_smash.pulp3.fixture_utils import add_recording_route
 
-from pulpcore.client.pulpcore import ApiClient, TasksApi
+from pulpcore.client.pulpcore import ApiClient, StatusApi, TasksApi
+from pulpcore.client.pulpcore.exceptions import ApiException
 
 
 cfg = get_config()
+cli = smash_cli.Client(cfg)
 SLEEP_TIME = _get_sleep_time(cfg)
+PULP_HOST = cfg.hosts[0]
+PULP_SERVICES = (
+    "pulpcore-content",
+    "pulpcore-api",
+    "new-pulpcore-worker@1",
+    "new-pulpcore-worker@2",
```

Comment:
I think we need to also have the regular worker names too in cases where the system under test is not the single container.